sal_interfaces/ATCamera/ATCamera_Events.xml

```diff
@@ -6464,18 +6464,189 @@
       <Count>1</Count>
     </item>
   </SALEvent>
-  <!--Created by xmlmaker2 for subsystem ATCamera class ATCamera_logevent_bonn_shutter_PeriodicTasks_timersSettingsApplied using level 1 for category timers-->
+  <!--Created by xmlmaker2 for subsystem ATCamera class ATCamera_logevent_bonn_shutter_PeriodicTasks_timersConfiguration using level 1 for category timers-->
   <!--======================================================================================================================================================-->
   <SALEvent>
     <Subsystem>ATCamera</Subsystem>
-    <EFDB_Topic>ATCamera_logevent_bonn_shutter_PeriodicTasks_timersSettingsApplied</EFDB_Topic>
+    <EFDB_Topic>ATCamera_logevent_bonn_shutter_PeriodicTasks_timersConfiguration</EFDB_Topic>
+    <item>
+      <EFDB_Name>version</EFDB_Name>
+      <Description>Version of the settings, formatted using CCS conventions</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>agentmonitorservice_taskPeriodMillis</EFDB_Name>
+      <Description>MISSING</Description>
+      <IDL_Type>long long</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>heartbeat_taskPeriodMillis</EFDB_Name>
+      <Description>MISSING</Description>
+      <IDL_Type>long long</IDL_Type>
+      <Units>unitless</Units>
```

Comment:
Shouldn't the attributes with `Millis` in the name have units as `second`?

Reply:
Yes, or I would think milliSecond. The units as well as the description comes from the CCS core, and currently they are missing there. There is a JIRA: https://jira.slac.stanford.edu/browse/LSSTCCS-2482 which is marked as resolved in core 6.0.3 but we have not yet switched to using that release.